barretenberg/acir_tests/flows/prove_then_verify_ultra_rollup_honk.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
can be deleted now that we have 1 singular prove_then_verify.sh file for plonk and honk